examples/user_interfaces/embedding_in_wx4.py

```diff
@@ -34,9 +36,15 @@ def __init__(self, canvas, cankill):
 
         # for simplicity I'm going to reuse a bitmap from wx, you'll
         # probably want to add your own.
-        self.AddSimpleTool(self.ON_CUSTOM, _load_bitmap('stock_left.xpm'),
-                           'Click me', 'Activate custom contol')
-        wx.EVT_TOOL(self, self.ON_CUSTOM, self._on_custom)
+        if 'phoenix' in wx.PlatformInfo:
+            self.AddTool(self.ON_CUSTOM, 'Click me',
+                         _load_bitmap('stock_left.xpm'),
+                         'Activate custom contol')
+            self.Bind(wx.EVT_TOOL, self._on_custom, id=self.ON_CUSTOM)
+        else:
+            self.AddSimpleTool(self.ON_CUSTOM, _load_bitmap('stock_left.xpm'),
+                               'Click me', 'Activate custom contol')
+            wx.EVT_TOOL(self, self.ON_CUSTOM, self._on_custom)
```

Comment:
similar comment here, although I don't know when AddTool() was added to the API.